sql-plugin/src/main/scala/org/apache/spark/sql/rapids/catalyst/expressions/GpuRandomExpressions.scala

```diff
@@ -52,7 +52,7 @@ case class GpuRand(child: Expression) extends ShimUnaryExpression with GpuExpres
   @transient protected lazy val seed: Long = child match {
     case GpuLiteral(s, IntegerType) => s.asInstanceOf[Int]
     case GpuLiteral(s, LongType) => s.asInstanceOf[Long]
-    case _ => throw new AnalysisException(
+    case _ => TrampolineUtil.throwAnalysisException(
```

Comment:
Why do we have a Trampoline method for throwing an AnalysisException _and_ a RapidsAnalysisException?  Seems like we should only have one of these, not both (probably just the latter).

Reply:
Looks like this was resolved by switching to RapidsAnalysisException.